gensim/test/test_ldamodel_distributed.py

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking transformation algorithms (the models package).
+"""
+
+
+import logging
+import unittest
+import numbers
+
+import six
+import numpy as np
+from numpy.testing import assert_allclose
+
+from gensim.corpora import mmcorpus, Dictionary
+from gensim.models import ldamodel, ldamulticore
+from gensim import matutils, utils
+from gensim.test import basetmtests
+from gensim.test.utils import datapath, get_tmpfile, common_texts
+import subprocess
+
+dictionary = Dictionary(common_texts)
+corpus = [dictionary.doc2bow(text) for text in common_texts]
+
+
+def testRandomState():
+    testcases = [np.random.seed(0), None, np.random.RandomState(0), 0]
+    for testcase in testcases:
+        assert(isinstance(utils.get_random_state(testcase), np.random.RandomState))
+
+
+class TestLdaModel(unittest.TestCase, basetmtests.TestBaseTopicModel):
+    def setUp(self):
+        self.corpus = mmcorpus.MmCorpus(datapath('testcorpus.mm'))
+        self.class_ = ldamodel.LdaModel
+        # test non-standard port to see if it still locates the right nameserver
+        self.host = '127.0.0.1'
+        self.port = 3123
+        env = {
+            'PYRO_SERIALIZERS_ACCEPTED': 'pickle',
+            'PYRO_SERIALIZER': 'pickle',
+        }
+        self.ns_process = Subprocess.Popen([sys.executable, '-m', 'Pyro4.naming', '--host=' + host, '--port=' + str(port)], env=env)
+        self.worker_processes = [
+            Subprocess.Popen([sys.executable, '-m', 'gensim.models.lsi_worker', '--host', host, '--port', str(port)], env=env)
+            for _ in range(2)
+        ]
+        self.dispatcher_process = Subprocess.Popen([sys.executable, '-m', 'gensim.models.lsi_dispatcher', '--host', host, '--port', str(port)], env=env)
+        self.model = self.class_(corpus, id2word=dictionary, num_topics=2, passes=100, distributed=True, ns_conf=dict(host=self.host, port=self.port))
+
+    def testTransform(self):
```

Comment:
Use PEP8 method names, here and everywhere.

```suggestion
    def test_transform(self):
```